backend/wmg/data/query.py

```diff
@@ -7,13 +7,14 @@
 
 from backend.wmg.data.snapshot import WmgSnapshot
 
-EMPTY_DIM_VALUES = ""
-
+# ASK madison if there's a better way to do this
+INDEXED_DIMENSIONS_TO_IGNORE = ["tissue_original_ontology_term_ids"]
```

Comment:
Looks like this isn't used anywhere?

Reply:
Oh yeah Madison suggested something else, will remove